### DISCUSSION TITLE
Character files changed from JSON to YAML

### DISCUSSION THREAD

**Discussion by theubie** [oobabooga/text-generation-webui#737] [General]
It might have been a good idea to talk with the community before making a major change like that.  Some people loathe the YAML format, plus the other UIs default to JSON.  

Yes, I understand you can load JSON with the same parser as YAML, but that's not the point.

**Comment by knoopx** [comment]
both formats seems to be supported, what's the issue?

**Comment by mcmonkey4eva** [comment]
JSON character files are still fully supported.

**Comment by lee-b** [comment]
YAML is a superset of JSON.  It literally does everything that JSON does, in the same syntax, and more.

**Reply by ThereforeGames** [reply]
Not quite true. [YAML does not parse tabs correctly](https://stackoverflow.com/questions/19975954/a-yaml-file-cannot-contain-tabs-as-indentation) while JSON does.

**Comment by Slug-Cat** [comment]
This breaks compatibility for transferring characters to most other places. Tavern and most websites I've seen do not handle YAML.

**Reply by mcmonkey4eva** [reply]
Should make feature requests or PRs to them then
